### PR TITLE
feat(verification/*): add verification package and unit tests for the…

### PR DESCRIPTION
**Pull request by drshrey**
… package

### CODE REVIEW

File: venv/.Python [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/Python
```
Comment (glemmaPaul) [inline venv/.Python]: Still here, please remove!

File: requirements-test.txt [hunk 2]
```diff
@@ -3,5 +3,6 @@ coverage
 mock>=1.0.1
 flake8>=2.1.0
 tox>=1.7.0
+twilio
```
Comment (glemmaPaul) [inline requirements-test.txt]: Maybe add the version number?

Comment (drshrey) [inline requirements-test.txt]: sounds good will do

**Comment by glemmaPaul** [comment]
LGTM

**Pull request merged**